tests/test_quotes.py

```diff
@@ -0,0 +1,35 @@
+from datetime import date
+from pathlib import Path
+
+import pandas as pd
+import yfinance
+from pytest import approx
+
+from investd.config import PERSIST_PATH
+from investd.quotes import adjust_symbol, generate_quotes_csv, load_quotes
+
+
+def test_adjust_symbol():
+    assert adjust_symbol("AAPL") == "AAPL"
+    assert adjust_symbol("BETASPXP.PL") == "BETASPXP.WA"
+    assert adjust_symbol("INR.FR") == "INR.PA"
+    assert adjust_symbol("CSPX.UK") == "CSPX.L"
+
+
+def test_generate_quotes_csv(monkeypatch, df_quotes: pd.DataFrame):
```

Comment:
you can add type annotation to monkeypatch as well

Reply:
Added it